### PR TITLE
Update Aragon Court FAQ

### PR DESCRIPTION
**Pull request by john-light**
- Expand Court governance documentation

### CODE REVIEW

File: user-guide/aragon-court/aragon-court-faq.html [hunk 1]
```diff
@@ -177,37 +177,103 @@ <h4 id="anj"><a href="#anj">ANJ</a></h4>
...
+				<li>Modules governor</li>
+			</ul>
+			</li>
+			<li>Module addresses:
+			<ul>
+				<li>Dispute Manager</li>
+				<li>Jurors Registry</li>
+				<li>Voting</li>
+				<li>Treasury</li>
+				<li>Subscriptions</li>
```
Comment (facuspagnuolo) [inline user-guide/aragon-court/aragon-court-faq.html:232]:
```suggestion
				<li>Subscriptions</li>
				<li>Set addresses for new modules</li>
```

File: user-guide/aragon-court/aragon-court-faq.html [hunk 2]
```diff
@@ -177,37 +177,103 @@ <h4 id="anj"><a href="#anj">ANJ</a></h4>
...
+		<li>Subscriptions module
+		<ul>
+			<li>Controller address (can only change by deploying a new Subscriptions module)</li>
+			<li>Period duration (can only change by deploying a new Subscriptions module)</li>
+			<li>Subscriptions fee token</li>
+			<li>Subscriptions fee amount</li>
+			<li>Number of pre-payment periods</li>
+			<li>Number of resume pre-paid periods</li>
+			<li>Governor share %</li>
+			<li>Late payment penalty %</li>
```
Comment (facuspagnuolo) [inline user-guide/aragon-court/aragon-court-faq.html:271]: Leaving a note here so we don't forget to update it after the upgrade

**Review by facuspagnuolo — APPROVED** [review]
LGTM!

File: user-guide/aragon-court/aragon-court-faq.html (hunk 1 above)
Comment (john-light) [reply user-guide/aragon-court/aragon-court-faq.html:232]: What is `<li>Set new module addresses</li>`? Is this an address with permission to "set new module addresses"? Or is this a boolean parameter, or something else...?

Comment (facuspagnuolo) [reply user-guide/aragon-court/aragon-court-faq.html:232]: It has the ability not just to change the existing modules' address, but to name new ones

Comment (john-light) [reply user-guide/aragon-court/aragon-court-faq.html:232]: New modules? So in addition to the five existing modules, more modules could be added? If so, "Set addresses for new modules" might be a better wording.

Comment (facuspagnuolo) [reply user-guide/aragon-court/aragon-court-faq.html:232]: Exactly, much better that way :)

**Pull request merged**